pkg/metrics/metrics.go

```diff
@@ -164,7 +164,12 @@ The label 'reason' can have the following values:
 - "PodsReadyTimeout" means that the eviction took place due to a PodsReady timeout.
 - "AdmissionCheck" means that the workload was evicted because at least one admission check transitioned to False.
 - "ClusterQueueStopped" means that the workload was evicted because the ClusterQueue is stopped.
-- "Deactivated" means that the workload was evicted because spec.active is set to false`,
+- "LocalQueueStopped" means that the workload was evicted because the LocalQueue is stopped.
+- "NodeFailures" means that the workload was evicted due to node failures when using TAS.
```

Comment:
```suggestion
- "NodeFailures" means that the workload was evicted due to node failures when using TopologyAwareScheduling.
```

I would propose using the complete name in case of a user-facing one.
We don't say AC, ProvReq, and MK. Instead of that, we want to say AdmissionCheck, ProvisioningRequest, and MultiKueue.

Reply:
done